test_runner/src/main/kotlin/ftl/args/ValidateCommonArgs.kt

```diff
@@ -72,3 +73,8 @@ fun IArgs.checkResultsDirUnique() {
     if (useLegacyJUnitResult && GcStorage.exist(resultsBucket, resultsDir))
         println("WARNING: Google cloud storage result directory should be unique, otherwise results from multiple test matrices will be overwritten or intermingled\n")
 }
+
+fun IArgs.checkDisableSharding() =
+    if (disableSharding && maxTestShards > 0)
+        println("WARNING: disable-sharding enabled with max-test-shards = $maxTestShards, Flank will ignore max-test-shard and disable sharding.")
+    else Unit
```

Comment:
do we need to return `Unit`

Reply:
if we do not return any value there should be 
```
fun IArgs.checkDisableSharding(){
if (disableSharding && maxTestShards > 0)
        println("WARNING: disable-sharding enabled with max-test-shards = $maxTestShards, Flank will ignore max-test-shard and disable sharding.")
    }
```

Reply:
We could also have util function for this, something similar to `check` or `require` (but without throwing an error)

Reply:
thanks, changed 👍